cmd/core/pfcp_connection.go

```diff
@@ -101,3 +114,42 @@ func (connection *PfcpConnection) SendMessage(msg message.Message, addr *net.UDP
 	}
 	return nil
 }
+
+func (connection *PfcpConnection) ProcessHeadlessAssociations() {
```

Comment:
Actually process every association. So I propose to name it like `UpdateAssociations`/`RefreshAssociations`